src/app/shared/shared-routing.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
`SharedModule` must not have `RoutingModule`, because it's contains only utilities `Components, Directives and Pipes`.